pkg/helpers/helpers.go

```diff
@@ -193,11 +193,23 @@ func CleanUpStaticObject(
 	case *rbacv1.RoleBinding:
 		err = client.RbacV1().RoleBindings(t.Namespace).Delete(ctx, t.Name, metav1.DeleteOptions{})
 	case *apiextensionsv1.CustomResourceDefinition:
-		err = apiExtensionClient.ApiextensionsV1().CustomResourceDefinitions().Delete(ctx, t.Name, metav1.DeleteOptions{})
```

Comment:
which case where the `apiExtensionClient` is set to be nil? Is this also possible for client?
I think it's better to check them at the beginning in the func, then we don't need to check in each case.